cookbook/core/in_container.mk

```diff
@@ -0,0 +1,24 @@
+SERIALIZED_PB_OUTPUT_DIR := /tmp/output
```

Comment:
Do all the subfolders have an `in_container.mk` file now?  If that's the case, what's the point of the top level `cookbook/in_container.mk`?